software_reseller/data/knowledge_article.xml

```diff
@@ -0,0 +1,27 @@
+            <field name="favorite_count">1</field>
+            <field name="is_locked" eval="True"/>
+            <field name="body">&lt;h1 data-last-history-steps="2686060328636411,2506460781552836,4216041616558028,1412267760541595,3717356699637718,1991048025115325,4074440199451603,476705159772834,3024777609364527,4469078993152238,3025487702027745,3861016984240661,995888713693056,3695863486355010,358377974321743,896909406774698,3028064440193867,2950199734246329,2547598295066870,2491792360742406,2992375316747207,3935074789772266,3950587467322961,322722864090148,4061615322632711,390443603933114,4370848648836978,1088142756277869,1033117873922130,3202166926663248,2405492542412764,4127715271861047,2920120766462179,1983212557212603,1565137919422218,3412735869464848,439980476406447,3763598801376470,2700704605872894,447092315142785,3321126000579521,3762543521192665,1207829724663820,3755403582505552,737119316659055,2302049048374568,2135546764405554,1585414501841915,867756304310765,2959091679830633,1180113052789649,4492148750628851,3108494955769139,1161653928097726,1121395452975578,3828872957291779,3310208256726571,89018768325882,704985900155000,851706572082219,2282829127291587,3943406677018411,886076624779135,3005628567386552,478784439851431,232865469713615,965281620599847,1295305467782148,3009107683343572,3481129057878095,4006931515915767,1509470257349715,950403516721277,1085392657786006,15501097608591,134775123691342,4424962527584263,1872831542162641,2798509990092869,714589608019715,2608463636688517,2685469717417151,1363053106131483,720041011496281,2738189766305012,1047857416831571,154562268731890,1379243647301730,2909947658034277,777798594018723,354662623399802,2632073360726844,1634974791409252,3261969914185556,1538303578692443,101168986152425,677990896199968,842995269378065,2296646545029035,2279611431807288,328012595176280,1596904697690510,2066249838370429,2221665958203665,2747154814883918,1374197696662776,436720216359837,1370416746923635,976943131896957,3798453871220178,3277240624553573,74890493043640,3935271506100724,631038053571141,3549552748886252,895334706304162,2219642061950249,3148813930248103,3255531770470923,1456089126597454,138062755690382,868865115410341,4196409524251838,2654018822660484,1495454557317230,770972886105756,4493484177739069,3211720378410254,2074389336212013,835727044607640,4128303608247807,665444987902266,408841611680148,2949999137375493,368339424617399,3490740284264613,364309742173738,1842429691716463,2815692436947435,363388517620128,1311561675502901,1874368935508069,3206330303678356,4396002721494888,3614143850094001,7207603093931,2144904362756241,4044111914757027,630869429415142,4243193191327627,2971693496708610,2524843682268222,3520826249415010,3326261570476748,2548948681798314,3124022637002555,2518921912251559,3287797742636994,1809228304181383,2033217485202654,2193463105647282,1758872550824538,79910072477425,28498934486387,653513432874377,506472268464903"&gt;
```

Comment:
can we remove the data-last-history-steps, as it is only there for collaborative mode, or at least empty it if we cannot remove it.